presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -401,16 +402,16 @@ private static String formatFragment(
 
         TypeProvider typeProvider = TypeProvider.fromVariables(fragment.getVariables());
         builder.append(
-                textLogicalPlan(
-                        fragment.getRoot(),
-                        typeProvider,
-                        Optional.of(fragment.getStageExecutionDescriptor()),
-                        functionAndTypeManager,
-                        fragment.getStatsAndCosts(),
-                        session,
-                        planNodeStats,
-                        1,
-                        verbose))
+                        textLogicalPlan(
+                                fragment.getRoot(),
+                                typeProvider,
+                                Optional.of(fragment.getStageExecutionDescriptor()),
+                                functionAndTypeManager,
+                                fragment.getStatsAndCosts(),
+                                session,
+                                planNodeStats,
+                                1,
+                                verbose))
```

Comment:
This are accidental changes that are (likely I guess) to break style checks. Please revert them back.